apps/backend/src/services/store.ts

```diff
@@ -0,0 +1,30 @@
+import { Store } from '../data/models/store';
+import { connection } from '../main';
+
+export const getAllStores = async (page: string) => {
+  try {
+    const pageNum = parseInt(page);
+
+    const stores = await connection
+      .getRepository(Store)
+      .createQueryBuilder('store')
+      .skip((pageNum - 1) * 10)
+      .take(10)
```

Comment:
maybe keep it configurable (value 10) or use from constants

Reply:
Agree. Will try to figure out how to implement it in the code in order to make it work fine. Thank you